docs/components/customization/user-button.mdx

```diff
@@ -227,3 +227,49 @@ With the above example, the `<UserButton />` menu items will be in the following
 1. Sign out
 1. Create organization
 1. Manage account
+
+## Conditionally render menu items
+
+To conditionally render menu items based on a user's role or permissions, you can use the `has()` helper function:
+
+```tsx filename="/app/page.tsx"
+'use client'
+
+import { UserButton, useAuth } from '@clerk/nextjs'
+
+const DotIcon = () => {
+  return (
+    <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512" fill="currentColor">
+      <path d="M256 512A256 256 0 1 0 256 0a256 256 0 1 0 0 512z" />
+    </svg>
+  )
+}
+
+export default function Home() {
+  const { has, isLoaded } = useAuth()
+
+  if (!isLoaded) {
+    return <span>Loading...</span>
+  }
+
+  const isAdmin = has({ role: 'org:admin' })
```

Comment:
We recommend PBAC > RBAC - so I'd like to see this use a permission instead please